.azure-pipelines/main.yml

```diff
@@ -8,6 +8,14 @@ resources:
 variables:
   Codeql.Enabled: true
 
+schedules:
```

Comment:
Can we only run this when there are actually new changes to build?

Reply:
That's how we end up with broken pipelines, it's happened so many times in the past.  I have most of the others running regularly.  If there's anything you suggest changing, like retention policy, etc., let me know, but we need a test build every so often.

Reply:
Sounds good, lets take it through then.